examples/rpc/starknet/starknet_getNonce.hurl

```diff
@@ -4,6 +4,7 @@ Content-Type: application/json
     "jsonrpc": "2.0",
     "method": "starknet_getNonce",
     "params": [
+        "latest",
```

Comment:
This looks to be positional:
`[next_inner] Deserialization to "starknet_ff::FieldElement" failed. Error: Error("invalid character", line: 0, column: 0), input JSON: "\n        \"latest\",\n        \"0x010b6c96d364cf182964fbd4a3438a5ae84cab990770c07994f9cb99fd26f6dc\"\n    ]"`

Reply:
I'm blocked by this PR, will add fix in mine